Source/WebCore/Modules/encryptedmedia/NavigatorEME.cpp

```diff
@@ -89,7 +89,7 @@ void NavigatorEME::requestMediaKeySystemAccess(Navigator& navigator, Document& d
 {
     // https://w3c.github.io/encrypted-media/#dom-navigator-requestmediakeysystemaccess
     // W3C Editor's Draft 09 November 2016
-    auto identifier = Logger::LogSiteIdentifier("NavigatorEME", __func__, &navigator);
+    auto identifier = Logger::LogSiteIdentifier("NavigatorEME"_s, __func__, &navigator);
```

Comment:
Too bad we can’t change `__func__` into an `ASCIILiteral`.

Reply:
One way would be a new macro which ends up calling `ASCIILiteral::fromLiteralUnsafe(__func__)` internally. Though then we'd have to come up with a good name.